client/state/ui/reducer.js

```diff
@@ -25,6 +26,31 @@ export function selectedSite( state = null, action ) {
 	return state;
 }
 
+export function section( state = false, action ) {
+	if ( action.type === SET_SECTION && action.section !== undefined ) {
```

Comment:
Because we have 3 reducers for the same `SET_SECTION` action, they are invoked only when user is feeding a isLoading / noSidebar / section
While I agree that they should be more granulated reducers, I honestly cannot tell from the old code calls that were `context.layout.setState ({section:, noSidebar, isLoading}) ` which call was intended.
I built in this safe-check to only reduce what was meant.
I opt to merge this and figure it out later.